### DISCUSSION TITLE
close button width

### DISCUSSION THREAD

**Discussion by Elementari** [mbnuqw/sidebery#1261] [Ideas]
Hello,
after spending some time with v5, one thing that I was missing most is the ability to define a width for the close button. In my setup, the close button is so much to the right, I wished I could make the close button wider which would make it easier to click. LIke adding # px to the left and right.

**Comment by zelch** [comment]
You can do this via the Styles editor.

The default CSS is:

```
.Tab .close-icon {
	position: absolute;
	top: calc(50% - 9px);
	left: calc(50% - 9px);
	width: 18px;
	height: 18px;
	fill: var(--tabs-normal-fg);
	opacity: .7;
}
```

Note, the left is 9px, exactly half of the width, this is important.

If you increase the width, you must ensure that the left remains half the value.

You will also need to add another rule:

```
.Tab .close {
  width: 18px;
}
```

That width must match the width in the first CSS section.

I'd also love to make the whole thing more visible, but I'm having some difficulty with that.
